packages/block-library/src/latest-posts/edit.js

```diff
@@ -27,15 +29,33 @@ import {
 } from '@wordpress/editor';
 import { withSelect } from '@wordpress/data';
 
+/**
+ * Module Constants
+ */
+const CATEGORIES_LIST_QUERY = {
+	per_page: 100,
+};
 const MAX_POSTS_COLUMNS = 6;
 
 class LatestPostsEdit extends Component {
 	constructor() {
 		super( ...arguments );
-
+		this.state = {
+			categoriesList: [],
+		};
 		this.toggleDisplayPostDate = this.toggleDisplayPostDate.bind( this );
 	}
 
+	componentWillMount() {
+		apiFetch( {
+			path: addQueryArgs( `/wp/v2/categories`, CATEGORIES_LIST_QUERY ),
```

Comment:
Doesn't this need to iterate the entire list? What if I have 102 categories?

Reply:
:woman_shrugging: this is just how it was in the original code, I was just fixing it so that it didn't break for authors

Reply:
Ok. I'll mention on #10873 and we can resolve with that.